daemon/network.go

```diff
@@ -358,6 +317,15 @@ func (daemon *Daemon) createNetwork(create types.NetworkCreateRequest, id string
 		nwOptions = append(nwOptions, libnetwork.NetworkOptionConfigFrom(create.ConfigFrom.Network))
 	}
 
+	if agent && driver == "overlay" && (create.Ingress || runtime.GOOS == "windows") {
```

Comment:
Why is Ingress network special here ?

Reply:
As mentioned above,  I treated ingress like other overlay networks with a load balancer sandbox since setting up and deleting the load balancer sandbox is the same as other networks.

So, on linux, if we are creating the ingress network, we go ahead and create the load balancer sandbox.  For all other overlay networks on linux, we don't do this.

Reply:
Same comment as above. I have to make sure this changes doesnt adversely impact Linux ingress network since the ingress-sbox is already managed in different code path. So, I would like to see if we can narrow down the scope of this change only to Windows without any impacts to Linux. We can remove the windows check once Linux makes full use of the LB-Sandbox concept.

But again, I have to spend a bit more time to understand the current changes made to ingress-sbox management in your previous ILB/ELB changes in to give a strong recommendation.

Reply:
I can limit the change to windows even more if you prefer.   It would just mean duplicate code for setting up the sandbox for ingress and other overlay networks on windows.  But, it would be less risky for linux.
Here is the previous code to setup the ingress-sbox (https://github.com/moby/moby/blob/9be245f438f9fb2eaeb7891673b16aed9262a192/daemon/network.go)

And here is the current code in libnetwork PR (https://github.com/docker/libnetwork/pull/2011/files#diff-cba6b1b3488fe48ebbef7e1821a3745dR2057)

Reply:
Thanks for the clarification. I think this change is fine as is.